src/components/item-icon.tsx

```diff
@@ -36,8 +38,8 @@ const ItemCircle: React.FC<GridProps> = props => (
   />
 );
 
-const StatusBubble: React.FC<any> = ({ tx }) => {
-  if (tx?.tx_status === 'pending') {
+const StatusBubble: React.FC<any> = ({ txStatus }) => {
```

Comment:
Also see this was done as `any` previously, but this'd be a good time to type properly